addons/web/static/tests/fields/relational_fields/field_one2many_tests.js

```diff
@@ -6598,15 +6591,9 @@ QUnit.module('fields', {}, function () {
 
             assert.containsOnce(form, '.o_data_row', "the one2many should contain one row");
 
-            // open the o2m record in readonly first
+            // open the o2m record (will open in edit)
             await testUtils.dom.click(form.$('.o_data_row td:first'));
-            assert.containsOnce(document.body, ".modal .o_form_readonly");
-            await testUtils.dom.click($('.modal .modal-footer .o_form_button_cancel'));
-
-            // switch to edit mode and open it again
-            await testUtils.form.clickEdit(form);
-            await testUtils.dom.click(form.$('.o_data_row td:first'));
```

Comment:
mouarf here you completely change the semantics of the test. Opening the form view first is important. Is there a way to keep opening it?